### PR TITLE
feat(Header): add logo

### PR DESCRIPTION
**Pull request by enddynayn**
Add new Frequency Logo that can be set
by setting the logoType prop to 'primary'.

Update Frequency Logo to adjust the fill color.

Screenshot 2024-10-30 at 11 24 51 AM

### CODE REVIEW

File: src/lib/assets/logos/FrequencyLogo.svelte [hunk 1]
```diff
@@ -8,6 +12,6 @@
...
-    fill="black"
+    fill={fillColor}
```
Comment (claireolmstead) [inline src/lib/assets/logos/FrequencyLogo.svelte:15]:
```suggestion
    fill="currentColor"
```
more general and reusable.

File: src/lib/assets/logos/FrequencyLogo.svelte [hunk 2]
```diff
@@ -1,3 +1,7 @@
+<script lang="ts">
+  export let fillColor = '#55B1AB';
+</script>
+
```
Comment (claireolmstead) [inline src/lib/assets/logos/FrequencyLogo.svelte:4]:
```suggestion
```

File: src/lib/features/Header.svelte [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <script lang="ts">
-  import { FrequencyAccessLogo, OpenClose } from '../assets/index';
+  import { FrequencyAccessLogo, FrequencyLogo, OpenClose } from '../assets/index';
   import NavMenu from './NavMenu.svelte';
   import NavMenuMobile from './NavMenuMobile.svelte';
   import { MenuItem } from '$lib/utils/types.js';
 
   export let menuItems: MenuItem[] = [];
-
+  export let logoType: 'primary' | 'access' = 'primary';
```
Comment (claireolmstead) [inline src/lib/features/Header.svelte:8]:
```suggestion
  export let logoType: 'primary' | 'access' = 'primary';
  
  export let logoColor = '#55B1AB'
```

File: src/lib/assets/logos/FrequencyLogo.svelte (hunk 1 above)
Comment (enddynayn) [reply src/lib/assets/logos/FrequencyLogo.svelte]: Nice!

**Review by claireolmstead — APPROVED** [review]
Paired and read through code. LGTM! 🚢

**Pull request merged**